github/templates/README.md

```diff
@@ -0,0 +1,79 @@
+PROJECT NAME
```

Comment:
I'm ok with this, but I don't think it's just a matter of taste, it has objectively less characters to write, and they will never change, if you want to keep it an `h1`.